src/System.Private.CoreLib/shared/System/Number.BigInteger.cs

```diff
@@ -470,10 +464,10 @@ public static uint CountSignificantBits(ulong value)
 
                 if (upper != 0)
                 {
-                    return 32 + CountSignificantBits(upper);
+                    return 32u + CountSignificantBits(upper);
```

Comment:
Does the `u` suffix make a difference here? I do not think we use it unless absolutely necessary (that is pretty much never).